benchmark/fs/bench-mkdirp-js.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const common = require('../common');
+const tmpdir = require('../../test/common/tmpdir');
+const mkdirp = require('../../deps/npm/node_modules/mkdirp');
```

Comment:
@maclover7 I believe @apapirovski was going to try benchmarking a few JavaScript implementations after this lands, with the proposal that he'd swap in the JavaScript implementation post-hoc if he could get the performance characteristics better --

made me think it might be worth leaving the benchmark in?

Reply:
I would suggest just turning this into one benchmark that runs the fs.mkdirp function and then I can use `benchmark-compare` to run it against the CPP version.